src/sql/parts/grid/views/gridParentComponent.ts

```diff
@@ -477,7 +477,13 @@ export abstract class GridParentComponent {
 	 */
 	xmlLinkHandler = (cellRef: string, row: number, dataContext: JSON, colDef: any) => {
 		const self = this;
-		self.handleLink(cellRef, row, dataContext, colDef, 'xml');
+
+		let value = self.getCellValueString(dataContext, colDef);
+		if (value.startsWith('<ShowPlanXML')) {
```

Comment:
Please also check that `column.columnName !== 'Microsoft SQL Server 2005 XML Showplan'` to avoid impacting the existing `Explain` experience.  We rename the display value of this column to `XML Showplan` which isn't as specific, but may be good enough to still support the desired `sp_whoisactive` without changing the XML link in `Explain`.